Compiling python3-tensorrt for Xavier error on dunfell

I'm trying to add python wrapper for tensorrt to dunfell latest release (R32.7.2/JP4.6.2).
During compile task I keep getting error as below

(just part of the log)
```
[2/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/parsers/pyCaffe.cpp.o -MF CMakeFiles/tensorrt.dir/src/parsers/pyCaffe.cpp.o.d -o CMakeFiles/tensorrt.dir/src/parsers/pyCaffe.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyCaffe.cpp
FAILED: CMakeFiles/tensorrt.dir/src/parsers/pyCaffe.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/parsers/pyCaffe.cpp.o -MF CMakeFiles/tensorrt.dir/src/parsers/pyCaffe.cpp.o.d -o CMakeFiles/tensorrt.dir/src/parsers/pyCaffe.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyCaffe.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyCaffe.cpp:20:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[3/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyAlgorithmSelector.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyAlgorithmSelector.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyAlgorithmSelector.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyAlgorithmSelector.cpp
FAILED: CMakeFiles/tensorrt.dir/src/infer/pyAlgorithmSelector.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyAlgorithmSelector.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyAlgorithmSelector.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyAlgorithmSelector.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyAlgorithmSelector.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyAlgorithmSelector.cpp:19:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[4/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/parsers/pyUff.cpp.o -MF CMakeFiles/tensorrt.dir/src/parsers/pyUff.cpp.o.d -o CMakeFiles/tensorrt.dir/src/parsers/pyUff.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyUff.cpp
FAILED: CMakeFiles/tensorrt.dir/src/parsers/pyUff.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/parsers/pyUff.cpp.o -MF CMakeFiles/tensorrt.dir/src/parsers/pyUff.cpp.o.d -o CMakeFiles/tensorrt.dir/src/parsers/pyUff.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyUff.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyUff.cpp:20:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[5/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyInt8.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyInt8.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyInt8.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyInt8.cpp
FAILED: CMakeFiles/tensorrt.dir/src/infer/pyInt8.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyInt8.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyInt8.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyInt8.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyInt8.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyInt8.cpp:20:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[6/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyFoundationalTypes.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyFoundationalTypes.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyFoundationalTypes.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyFoundationalTypes.cpp
FAILED: CMakeFiles/tensorrt.dir/src/infer/pyFoundationalTypes.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyFoundationalTypes.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyFoundationalTypes.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyFoundationalTypes.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyFoundationalTypes.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyFoundationalTypes.cpp:19:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[7/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/parsers/pyOnnx.cpp.o -MF CMakeFiles/tensorrt.dir/src/parsers/pyOnnx.cpp.o.d -o CMakeFiles/tensorrt.dir/src/parsers/pyOnnx.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyOnnx.cpp
FAILED: CMakeFiles/tensorrt.dir/src/parsers/pyOnnx.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/parsers/pyOnnx.cpp.o -MF CMakeFiles/tensorrt.dir/src/parsers/pyOnnx.cpp.o.d -o CMakeFiles/tensorrt.dir/src/parsers/pyOnnx.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyOnnx.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/parsers/pyOnnx.cpp:20:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[8/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyCore.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyCore.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyCore.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyCore.cpp
FAILED: CMakeFiles/tensorrt.dir/src/infer/pyCore.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyCore.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyCore.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyCore.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyCore.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyCore.cpp:19:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[9/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/pyTensorRT.cpp.o -MF CMakeFiles/tensorrt.dir/src/pyTensorRT.cpp.o.d -o CMakeFiles/tensorrt.dir/src/pyTensorRT.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/pyTensorRT.cpp
[10/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyGraph.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyGraph.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyGraph.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyGraph.cpp
FAILED: CMakeFiles/tensorrt.dir/src/infer/pyGraph.cpp.o 
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyGraph.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyGraph.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyGraph.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyGraph.cpp
In file included from /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyGraph.cpp:19:0:
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h: In function 'pybind11::function tensorrt::utils::getOverride(const T*, const string&, bool)':
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: error: 'get_override' is not a member of 'tensorrt::utils::py'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include/utils.h:114:33: note: suggested alternative: 'get_overload'
     py::function overload = py::get_override(self, overloadName.c_str());
                                 ^~~~~~~~~~~~
                                 get_overload
[11/12] /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/bin/aarch64-poky-linux/aarch64-poky-linux-g++  -Dtensorrt_EXPORTS -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot/usr/include/python3.8 -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/../include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/include -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/docstrings -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot  -O2 -pipe -g -feliminate-unused-debug-types -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0=/usr/src/debug/python3-tensorrt/8.2.1-r0                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot=                           -fdebug-prefix-map=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native=  -fvisibility-inlines-hidden  -march=armv8-a+crc -fstack-protector-strong  -D_FORTIFY_SOURCE=2 -Wformat -Wformat-security -Werror=format-security  --sysroot=/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot -I/home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/recipe-sysroot-native/usr/local/cuda-10.2/include -I=/usr/local/cuda-10.2/include -fvisibility=hidden -std=c++11 -Wno-deprecated-declarations -DNDEBUG -fPIC   -std=gnu++11 -MD -MT CMakeFiles/tensorrt.dir/src/infer/pyPlugin.cpp.o -MF CMakeFiles/tensorrt.dir/src/infer/pyPlugin.cpp.o.d -o CMakeFiles/tensorrt.dir/src/infer/pyPlugin.cpp.o -c /home/ubuntu/przemoch/safeway-by-sternkraft/build/tmp/work/armv8a_tegra194-poky-linux/python3-tensorrt/8.2.1-r0/git/python/src/infer/pyPlugin.cpp
ninja: build stopped: subcommand failed.
WARNING: exit code 1 from a shell command.
```

I took recipe from external/meta-python.

Anyone has idea how to resolve it?

--- a/external/meta-python/recipes-devtools/python3-tensorrt/python3-tensorrt/0001-Fixups-for-cross-building-in-OE.patch
+++ b/external/meta-python/recipes-devtools/python3-tensorrt/python3-tensorrt/0001-Fixups-for-cross-building-in-OE.patch
@@ -1,4 +1,4 @@
-From 800ade095be320586162e08fe8dc94db56f8493f Mon Sep 17 00:00:00 2001
+From 3b39935086348e02389480e260e3bc4b14712a15 Mon Sep 17 00:00:00 2001
 From: Matt Madison <matt@madison.systems>
 Date: Fri, 11 Mar 2022 11:20:55 -0800
 Subject: [PATCH] Fixups for cross building in OE
@@ -6,7 +6,8 @@ Subject: [PATCH] Fixups for cross building in OE
 ---
  python/CMakeLists.txt                | 9 ++-------
  python/include/ForwardDeclarations.h | 2 +-
- 2 files changed, 3 insertions(+), 8 deletions(-)
+ python/include/utils.h               | 2 +-
+ 3 files changed, 4 insertions(+), 9 deletions(-)
 
 diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
 index 5672db9..5f46c90 100644
@@ -55,6 +56,16 @@ index d9d891a..9a6c7a7 100644
  
  // We need to avoid making copies of PluginField because it does not own any of it's members.
  // When there are multiple PluginFields pointing to the same data in Python, bad things happen.
--- 
-2.32.0
-
+diff --git a/python/include/utils.h b/python/include/utils.h
+index f1d3a2d..6847b9a 100644
+--- a/python/include/utils.h
++++ b/python/include/utils.h
+@@ -111,7 +111,7 @@ inline size_t volume(const nvinfer1::Dims& dims)
+ template <typename T>
+ py::function getOverride(const T* self, const std::string& overloadName, bool showWarning = true)
+ {
+-    py::function overload = py::get_override(self, overloadName.c_str());
++    py::function overload = py::get_overload(self, overloadName.c_str());
+     if (!overload && showWarning)
+     {
+         std::cerr << "Method: " << overloadName

